src/Display.h

```diff
@@ -2,24 +2,23 @@
 #define DISPLAY_H
 
 #include <string>
+
 #include <SDL2/SDL.h>
 
 class Display
 {
 public:
 	Display(int width, int height, const std::string& title);
+	virtual ~Display();
+
 	void Update();
 	bool IsClosed();
 	void Clear(float red, float green, float blue, float alpha);
-	virtual ~Display();
-protected:
-private:
-	Display(const Display& other);
-	void operator=(const Display& other) {}
```

Comment:
It is good practice to leave the assignment operator disabled. What I recommend is using c++11 syntax and doing 
```c++
void operator=(const Display& other) = delete
```
(as a public member). 

Or you could add it back in as it was.